cstrml/payment/src/mock.rs

```diff
@@ -134,14 +134,15 @@ impl balances::Trait for Test {
 }
 
 impl tee::Trait for Test {
+    type Currency = Balances;
```

Comment:
it is not wise for putting Currency into tee just for the *MarketInterface*. We may reconstruct this in the next PR.